test/catch2/block/util_block.cpp

```diff
@@ -103,7 +103,6 @@ create_write_buffer(WT_BM *bm, std::shared_ptr<mock_session> session, std::strin
     test_and_validate_write_size(bm, session, buf_memsize, allocation_size);
 
     // Initialize the buffer with aligned size.
```

Comment:
Should this comment be updated accordingly, or removed?

Reply:
The function above `test_and_validate_write_size` checks for the alignment so the comment still makes sense.

Reply:
Thanks for confirming!